bin/sce_to_anndata.R

```diff
@@ -44,5 +55,29 @@ sce <- readr::read_rds(opt$input_sce_file)
 # export sce as anndata object
 scpcaTools::sce_to_anndata(
   sce,
-  anndata_file = opt$output_h5_file
+  anndata_file = opt$output_rna_h5
 )
+
+# if feature data exists, grab it and export to AnnData 
+if(!is.null(opt$feature_name)){
+  
+  # make sure the feature data is present 
+  if(!(opt$feature_name %in% altExpNames(sce))){
+    stop("feature_name must match name of altExp in provided SCE object.")
+  }
+  
+  # check for output file 
+  if(!(stringr::str_ends(opt$output_feat_h5, ".hdf5|.h5"))){
```

Comment:
following earlier suggestion
```suggestion
  if(!(stringr::str_ends(opt$output_feature_h5, ".hdf5|.h5"))){
```